src/dotnet-watch/CommandLineOptions.cs

```diff
@@ -53,6 +53,10 @@ public static CommandLineOptions Parse(string[] args, IConsole console)
   DOTNET_WATCH
   dotnet-watch sets this variable to '1' on all child processes launched.
 
+  DOTNET_WATCH_ITERATION
+  dotnet-watch sets this variable to '0' and increments by one each time
+  the inner command is restarted.
```

Comment:
"Inner command"?  Docs leaking implementation details?

Reply:
What about this?

> dotnet-watch sets this variable to '1' and increments by one each time a file is changed and the command is restarted.